params/types/goethereum/goethereum_configurator.go

```diff
@@ -691,6 +703,13 @@ func (c *ChainConfig) IsEnabled(fn func() *uint64, n *big.Int) bool {
 	if f == nil || n == nil {
 		return false
 	}
+	fnName := runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name()
+	if strings.Contains(fnName, "ECBP1100Transition") {
+		disabledTransition := c.GetECBP1100DeactivateTransition()
```

Comment:
@meowsbits  this is the last one. `s/disabledTransition/deactivateTransition/g`

Reply:
Danke, done and force pushed again (squashed into https://github.com/etclabscore/core-geth/pull/592/commits/7a026c079bf33bceca4629cb8132628f99baad41)